COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -913,11 +920,13 @@ namespace COLLADASaxFWL
                 mColorList.push_back ( color );
             }
         }
+		return false;
     }
 
     //------------------------------
-    void MeshLoader::initializeTexCoordsOffset ()
+    bool MeshLoader::initializeTexCoordsOffset ()
     {
+		bool has_errors = false;
```

Comment:
its quite simple, you just have to push changes in your own branch (gaiaclary:improved_parser_abort_on_error) , the one referred by this PR, and it will automatically be updated.